common/src/commonMain/kotlin/io/xorum/codeforceswatcher/network/KtorResponseClient.kt

```diff
@@ -1,21 +1,30 @@
 package io.xorum.codeforceswatcher.network
 
-import io.ktor.client.features.ClientRequestException
-import io.ktor.client.request.HttpRequestBuilder
-import io.ktor.client.request.get
-import io.ktor.client.request.post
-import io.ktor.client.request.put
+import io.ktor.client.*
+import io.ktor.client.features.*
+import io.ktor.client.features.json.*
+import io.ktor.client.features.json.serializer.*
+import io.ktor.client.features.logging.*
+import io.ktor.client.request.*
+import io.ktor.http.*
 import io.ktor.utils.io.ByteReadChannel
 import io.ktor.utils.io.readUTF8Line
 import io.xorum.codeforceswatcher.network.responses.Response
 import io.xorum.codeforceswatcher.redux.analyticsController
+import io.xorum.codeforceswatcher.redux.getLang
+import io.xorum.codeforceswatcher.redux.store
 import io.xorum.codeforceswatcher.util.stringify
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.json.Json
 
+const val BACKEND_PROD_LINK = "algoris-me-backend.herokuapp.com"
+const val BACKEND_STAGING_LINK = "algoris-me-backend-staging.herokuapp.com"
+
+lateinit var backendLink: String
+
 class KtorResponseClient {
 
-    val backendApiClient by lazy { makeBackendApiClient() }
+    val backendApiClient = makeBackendApiClient()
```

Comment:
I'm not sure this will work as expected in all cases. Why don't you want to have `lazy`?